torch/_export/serde/schema.py

```diff
@@ -359,6 +359,7 @@ class GraphModule:
     # the modules in order to unflatten the modules back to the eager calling
     # conventions.
     module_call_graph: List[ModuleCallEntry]
+    disable_forced_specializations: bool = False
```

Comment:
This feels very wrong. I don't think we should get rid of checks here, but rather try converting them to real checks (so IR doesn't have this field). Or just not do any checks at all if this flag is `True`, in which rename the flag to `disable_runtime_assertions`.